src/lib/Form/Data/Content/ContentUpdateMainLocationData.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Data\Content;
+
+use eZ\Publish\API\Repository\Values\Content\Location;
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @todo Add validation.
+ */
+class ContentUpdateMainLocationData
```

Comment:
Name doesn't follow our notation, correct name should be `ContentMainLocationUpdateData`.